hyperwallet/tests/test_api.py

```diff
@@ -2267,5 +2267,34 @@ def test_create_transfer_spend_back_refunds_success(self, mock_get):
 
         self.assertTrue(response.token, self.data.get('token'))
 
+    '''
+
+    Create User Status Transition
+
+    '''
+
+    def test_create_user_status_transition_user_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.createUserStatusTransition()
+
```

Comment:
Remove the blank line. Remove all unwanted blank lines.

Reply:
as per your comments has been resolved